### PR TITLE
Taginputlabelposition

### PR DESCRIPTION
**Pull request by luisarruda**
## Proposed Changes

Fix the label position on the taginput when using maxtags with label-position setted to inside or on-border

### CODE REVIEW

**Comment by luisarruda** [comment]
This is what happens on the current version. 
IMG_20191009_005617_500
IMG_20191009_005615_219

**Comment by jtommy** [comment]
@luisarruda Thanks but i'm checking it and there is an issue using two or more taginput below 
image

**Comment by luisarruda** [comment]
@jtommy Sorry, I didn't notice that, now I change the ".help.counter" the input use this class too, but the change does not affect it. (on my tests)

File: src/scss/components/_form.scss [hunk 1]
```diff
@@ -242,8 +242,9 @@ $floating-in-height: 3.25em !default;
 
 .control {
     .help.counter {
-        float: right;
+        float: none;
```
Comment (jtommy) [inline src/scss/components/_form.scss]: Please remove it!

Comment (luisarruda) [reply src/scss/components/_form.scss]: Ok, done. Because the default value of the property is none?

Comment (jtommy) [reply src/scss/components/_form.scss]: Sure

Comment (jtommy) [reply src/scss/components/_form.scss]: Yes, but i'm checking and float: right attribute removes the top margin, so you should restore it

**Comment by jtommy** [comment]
@luisarruda i fixed it by https://github.com/buefy/buefy/commit/eac754819b29114556990844a5132025bfcf9b86, btw thanks for the tip

**Pull request closed without merging**